BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -596,6 +596,17 @@ Some devices have hardware support for AES-CCM but not AES-GCM, so they are unab
 
 When using ECDSA signatures for authentication of TLS peers, it is RECOMMENDED that implementations use the NIST curve P-256. In addition, to avoid predictable or repeated nonces (that would allow revealing the long term signing key), it is RECOMMENDED that implementations implement "deterministic ECDSA" as specified in {{!RFC6979}} and in line with the recommendations in {{RFC8446}}.
 
+Note that implementations of "deterministic ECDSA" may be vulnerable to certain
+side-channel and fault injection attacks precisely because of their
+determinism.  While most fault attacks described in the literature assume
+physical access to the device (and therefore are more relevant in IoT
+deployments with poor or non-existent physical security), some can be carried
+out remotely, e.g., as Rowhammer variants.  In deployments where side-channel
```

Comment:
Need a ref for Rowhammer, e.g. https://www.blackhat.com/docs/us-15/materials/us-15-Seaborn-Exploiting-The-DRAM-Rowhammer-Bug-To-Gain-Kernel-Privileges.pdf or https://nakedsecurity.sophos.com/2015/03/12/row-hammering-how-to-exploit-a-computer-by-overworking-its-memory/

Reply:
> Need a ref for Rowhammer

Good point!

> e.g. https://www.blackhat.com/docs/us-15/materials/us-15-Seaborn-Exploiting-The-DRAM-Rowhammer-Bug-To-Gain-Kernel-Privileges.pdf or https://nakedsecurity.sophos.com/2015/03/12/row-hammering-how-to-exploit-a-computer-by-overworking-its-memory/

I think we should use the [original row hammer paper](https://users.ece.cmu.edu/~yoonguk/papers/kim-isca14.pdf) for the class and / or [Poddebniak's](https://eprint.iacr.org/2017/1014.pdf) as a specific instance attacking deterministic EC though.

Reply:
I would go for Poddebniak.

Reply:
oops, I've done both in 6a1d8a80